lib/msal-core/package.json

```diff
@@ -37,14 +37,17 @@
   },
   "dependencies": {
     "js-base64": "^2.5.1",
-    "tslib": "^1.9.3"
+    "sha256": "^0.2.0",
```

Comment:
I might consider a different sha-256 library here. I know it's a small thing but encryption libraries are notorious for being attack vectors. It worries me that this only has three published versions, and I also don't see any developers that I can trust here. Something like [js-crypto](https://www.npmjs.com/package/crypto-js) or [Stanford's Javascript Crypto Library](https://www.npmjs.com/package/sjcl) might be a more trustworthy option since they have more releases and trusted developers behind them.

Reply:
uninstalling for now